src/apps/Altinn.AccessManagement/src/Altinn.AccessMgmt.Persistence/Definitions/RelationDefinition.cs

```diff
@@ -67,10 +193,40 @@ UNION ALL
         fa.toid            AS viaid,
         ta.roleid          AS viaroleid,
         ta.toid,
+        dp.packageid,
+        null::uuid as resourceid,
         'Delegation'::text AS reason
         FROM dbo.delegation d
         JOIN dbo.assignment fa ON fa.id = d.fromid
         JOIN dbo.assignment ta ON ta.id = d.toid
+        JOIN dbo.delegationpackage dp ON dp.delegationid = d.id
+
+        UNION ALL
+
+        SELECT fa.fromid,
+        fa.roleid,
+        fa.toid            AS viaid,
+        ta.roleid          AS viaroleid,
+        ta.toid,
+        null::uuid as packageid,
+        dr.resourceid,
+        'Delegation'::text AS reason
+        FROM dbo.delegation d
+        JOIN dbo.assignment fa ON fa.id = d.fromid
+        JOIN dbo.assignment ta ON ta.id = d.toid
+        JOIN dbo.delegationresource dr ON dr.delegationid = d.id
```

Comment:
DelegationResource er ikke noe vi har eller kommer til å ha i nær fremtid så kan droppes for nå i stedet for å gjøre en unødvendig spørring